docs/content/contribute/index.md

```diff
@@ -1,48 +1,109 @@
+## 🛠️ Setting up the Docs
 
-    ```shell
-    python -m pip install -r docs/requirements.txt
-    ```
+### Prerequisites
 
-    or
+- Git installed on your machine
+- Python 3.8 or higher
+- A text editor of your choice
 
-    ```shell
-    python3 -m pip install -r docs/requirements.txt
-    ```
+### Step-by-Step Setup Guide
 
-1. Start iterating using mkdocs' live server:
+1. **Clone the Repository** 📥
+   ```shell
+   git clone github.com/Significant-Gravitas/AutoGPT.git
+   ```
 
-    ```shell
-    mkdocs serve
-    ```
+2. **Install Dependencies** 📚
+   ```shell
+   python -m pip install -r docs/requirements.txt
+   ```
+   or
+   ```shell
+   python3 -m pip install -r docs/requirements.txt
+   ```
 
-1. Open your browser and navigate to `http://127.0.0.1:8000`.
+3. **Start the Development Server** 🔥
+   ```shell
+   mkdocs serve
+   ```
 
-1. The server will automatically reload the docs when you save your changes.
+4. **View the Docs** 🌐
+   - Open your browser
+   - Navigate to `http://127.0.0.1:8000`
+   - Changes will auto-reload! 🔄
 
-## Adding a new page
+## 📝 Adding New Content
 
-1. Create a new markdown file in the `docs/content` directory.
-1. Add the new page to the `nav` section in the `mkdocs.yml` file.
-1. Add the content to the new markdown file.
-1. Run `mkdocs serve` to see your changes.
+### Creating a New Page
 
-## Checking links
+1. **Create the File** 📄
+   - Navigate to `docs/content`
+   - Create a new markdown file
 
-To check for broken links in the documentation, run `mkdocs build` and look for warnings in the console output.
+2. **Update Navigation** 🗺️
+   - Open `mkdocs.yml`
+   - Add your page to the `nav` section
 
-## Submitting a Pull Request
+3. **Add Content** ✍️
+   - Write your content using Markdown
+   - Add images and code examples as needed
 
-When you're ready to submit your changes, please create a pull request. We will review your changes and merge them if they are appropriate.
+4. **Preview Changes** 👀
+   - Run `mkdocs serve`
+   - Check your new page in the browser
+
+## 🔍 Quality Checks
+
+### Link Validation
+```shell
+mkdocs build
+```
+Watch for any warnings about broken links in the console output! 🚨
+
+## 🎯 Submitting Your Contribution
+
+### Pull Request Process
+
+1. **Create a Branch** 🌿
+   - Use a descriptive name for your branch
+   - Keep changes focused and atomic
+
+2. **Submit PR** 📮
+   - Fill out the PR template
+   - Add screenshots if applicable
+   - Link related issues
+
+3. **Review Process** 👥
+   - Address reviewer feedback
+   - Make requested changes
+   - Maintain active communication
+
+## ⭐ Best Practices
+
+- Keep documentation clear and concise
+- Use proper Markdown formatting
+- Include code examples where helpful
+- Test all links before submitting
+- Follow existing documentation style
+
+---
+
+<div align="center">
+
+**Thank you for contributing to our documentation!** ❤️
+
+*Together, we make documentation better for everyone.*
```

Comment:
I'm not sure why this isn't bolded correctly
![image](https://github.com/user-attachments/assets/fb59826a-2e0c-416b-9788-78765e8cf2f5)